packages/core/upload/admin/src/components/AssetDialog/index.js

```diff
@@ -20,11 +21,15 @@ import { useFolders } from '../../hooks/useFolders';
 import useModalQueryParams from '../../hooks/useModalQueryParams';
 import { AssetDefinition } from '../../constants';
 import getAllowedFiles from '../../utils/getAllowedFiles';
-import { DialogTitle } from './DialogTitle';
+import { DialogHeader } from './DialogHeader';
 import { DialogFooter } from './DialogFooter';
 import { EditAssetDialog } from '../EditAssetDialog';
 import { moveElement } from '../../utils/moveElement';
 
+const LoadingBody = styled(Flex)`
+  min-height: calc(60vh + 80px);
```

Comment:
Thanks for your explanation, which makes sense to me. Could you add a code comment explaining, where this magic number of `80` comes from? I find it very useful to understand later :)

Reply:
of course!

Reply:
Also: since everything else is `rem` this should probably be too, because if the font-size of the user changes, it might affect the height of the tabs.

Reply:
mmh I see, but since the ModalBody is using `60vh`, wouldn't we have different responsiveness between the loading modal and the other ones?

Reply:
I tested `60rem` instead of `vh`, the result wasn't looking good
At least I replaced the `80px` by `pxToRem(80)` and added a comment, ready for your review :)